app/src/main/java/org/oppia/app/topic/overview/TopicOverviewViewModel.kt

```diff
@@ -0,0 +1,27 @@
+package org.oppia.app.topic.overview
+
+import android.widget.ImageView
+import androidx.databinding.BindingAdapter
+import androidx.databinding.ObservableField
+import androidx.lifecycle.ViewModel
+import org.oppia.app.R
+import org.oppia.app.fragment.FragmentScope
+import org.oppia.app.model.Topic
+import org.oppia.app.viewmodel.ObservableViewModel
+import javax.inject.Inject
+
+/** [ViewModel] for showing topic overview details. */
+@FragmentScope
+class TopicOverviewViewModel @Inject constructor() : ObservableViewModel() {
+  companion object {
+    @JvmStatic
+    @BindingAdapter("android:src")
+    fun setBackgroundResource(downloadStatus: ImageView, resource: Int) {
```

Comment:
Suggest using the adapter introduced in #195 for automatically binding story thumbnails to an ImageView for simplicity.

Reply:
Used `ImageViewBindingAdapters` and removed this `companion object` from this file.